### DISCUSSION TITLE
Bottom of the Leaderboard shows converted scores with 0 points

### DISCUSSION THREAD

**Discussion by marcostudios** [ppy/osu#24655] [Q&A] [answered]
At the bottom of the leaderboards, some converted scores show 0 points, why is it happening?

![immagine](https://github.com/ppy/osu/assets/5209299/4914c655-64d1-4d54-9549-520ee62f15bc)

![immagine](https://github.com/ppy/osu/assets/5209299/33a7d597-0ef8-4fab-b45b-37d02794aa61)

![immagine](https://github.com/ppy/osu/assets/5209299/8db5f41b-e456-4678-a6aa-57def52829e2)

**Comment by Theighlin** [comment, marked as answer]
Tracking at #24663
